src/modules/orders/selectors/filled-orders.js

```diff
@@ -24,15 +24,14 @@ function findOrders(filledOrders, accountId, outcomesData, marketsData) {
       let typeOp = type;
 
       const outcomeName =
-        outcomesData[marketId][outcome].name ||
-        outcomesData[marketId][outcome].name;
+        outcomesData[outcome].name || outcomesData[outcome].name;
```

Comment:
Not sure what this is? same property || same property, if I'm reading this correctly.

Reply:
that is left over from when scalar names were done in there. will remove